pkg/rancher/rancher.go

```diff
@@ -196,14 +197,20 @@ func New(ctx context.Context, clientConfg clientcmd.ClientConfig, opts *Options)
 		return nil, err
 	}
 
+	extensionAPIServer, err := ext.NewExtensionAPIServer(wranglerContext)
+	if err != nil {
+		return nil, fmt.Errorf("extension api server: %w", err)
+	}
+
 	steve, err := steveserver.New(ctx, restConfig, &steveserver.Options{
-		ServerVersion:   settings.ServerVersion.Get(),
-		Controllers:     steveControllers,
-		AccessSetLookup: wranglerContext.ASL,
-		AuthMiddleware:  steveauth.ExistingContext,
-		Next:            ui.New(wranglerContext.Mgmt.Preference().Cache(), wranglerContext.Mgmt.ClusterRegistrationToken().Cache()),
-		ClusterRegistry: opts.ClusterRegistry,
-		SQLCache:        features.UISQLCache.Enabled(),
+		ServerVersion:      settings.ServerVersion.Get(),
+		Controllers:        steveControllers,
+		AccessSetLookup:    wranglerContext.ASL,
+		AuthMiddleware:     steveauth.ExistingContext,
+		Next:               ui.New(wranglerContext.Mgmt.Preference().Cache(), wranglerContext.Mgmt.ClusterRegistrationToken().Cache()),
+		ClusterRegistry:    opts.ClusterRegistry,
+		SQLCache:           features.UISQLCache.Enabled(),
+		ExtensionAPIServer: extensionAPIServer,
```

Comment:
ExtensionAPIServer can be nil in which case it will not be served (404 will be served).